templates/aws/outposts.html

```diff
@@ -1,202 +1,374 @@
 {% extends "aws/base_aws.html" %}
 
 {% block title %}Ubuntu Pro & Ubuntu on AWS Outposts{% endblock %}
-{% block meta_description %}The most popular Operating System in the public cloud, running on your premises with a consistent AWS experience. {% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1aXtrGHw7AcZ_qrov3kJahRtgsmbNx7eH5aWFlIySpfE/edit#{% endblock meta_copydoc %}
+
+{% block meta_description %}
+  The most popular Operating System in the public cloud, running on your premises with a consistent AWS experience.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1aXtrGHw7AcZ_qrov3kJahRtgsmbNx7eH5aWFlIySpfE/edit#
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
-  <div class="p-strip--suru-topped">
-    <div class="row u-equal-height">
-      <div class="col-7 col-medium-4">
-        <h1>Ubuntu on AWS Outposts</h1>
-        <p>The most popular operating system in the public cloud, running on your premises with a consistent AWS experience.</p>
-        <p>Security, compliance and support available for all enterprise scenarios.</p>
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <div class="p-section--shallow">
+          <h1>Ubuntu on AWS Outposts</h1>
+          <p>
+            The most popular operating system in AWS, running on your premises with a consistent hybrid
+            experience.
+          </p>
+          <p>Security, compliance and support available for all enterprise scenarios.</p>
+        </div>
+        <hr class="is-muted" />
```

Comment:
This is being depreciated so now `p-rule--muted` is preferred over `is-muted`. (for all horizontal rules)
This might cause the space issues but we have that being fixed in Vanilla